docs/deploy/providers/cloudflare.md

```diff
@@ -19,8 +19,10 @@ bucket = ".output/public"
 entry-point = ".output"
 
 [build]
-command = "true"
-upload.format = "service-worker"
+command = "yarn build"
```

Comment:
We assume `.output` is already built in github actions (this is important because `NITRO_PRESET="cloudflare"` should be set for the correct format.

Reply:
I'm not sure about this - depends on where `wrangler publish` is run so it can point to the correct entry.

Reply:
Are you exactly following guide in the [docs](https://nitro.unjs.io/deploy/providers/cloudflare.html#deploy-from-your-local-machine-using-wrangler) or using your own action / CI-CD workflow?

Reply:
Yes following the guide does not work. Logs posted in the comment below.

Reply:
(the guide is not meant for windows PC but github action with linux)